### PR TITLE
Added Swift Package Manager support

### PR DESCRIPTION
**Pull request by RomanTysiachnik**
## Description
- Added `Package.swift` for SPM support;
- Fixed issues in `.xib` files;
- Updated `Readme.md` with instructions for Swift Package Manager.

### CODE REVIEW

**Comment by RomanTysiachnik** [comment]
@ocollet FYI

File: UnsplashPhotoPicker/UnsplashPhotoPicker/Classes/Views/CheckmarkView.swift [hunk 1]
```diff
@@ -14,8 +14,13 @@ class CheckmarkView: UIView {
     override var intrinsicContentSize: CGSize { return CGSize(width: 24, height: 24) }
 
     private lazy var checkmark: UIImageView = {
-        let bundle = Bundle(for: type(of: self))
-        let image = UIImage(named: "checkmark", in: bundle, compatibleWith: nil)
+        let bundle = Bundle.local
+        let image: UIImage?
+        if #available(iOS 13, *) {
+          image = UIImage(systemName: "checkmark.circle")
```
Comment (ocollet) [inline UnsplashPhotoPicker/UnsplashPhotoPicker/Classes/Views/CheckmarkView.swift:20]: The SF Symbol icon doesn't look good. If anything, it should be in a separate PR. Can you revert this change?

**Bad**
SF Symbol - Bad

**Good**
Custom image - Good

**Review by ocollet — CHANGES_REQUESTED** [review]
Can you revert the checkmark icon changes?
Everything else is good. 👍

**Comment by Kondamon** [comment]
@RomanTysiachnik may I kindly ask if you could please remove the requested 2 lines from the review? So this pull request can be merged! It is prohibited to edit your pull request, otherwise I would have done. Thank you!

File: UnsplashPhotoPicker/UnsplashPhotoPicker/Classes/Views/CheckmarkView.swift (hunk 1 above)
Comment (RomanTysiachnik) [reply UnsplashPhotoPicker/UnsplashPhotoPicker/Classes/Views/CheckmarkView.swift:20]: @ocollet Yeah, sure!

**Comment by RomanTysiachnik** [comment]
@ocollet @Kondamon ready!

**Review by ocollet — APPROVED** [review]
Looks good, thanks! 👍

**Pull request merged**